### PR TITLE
fix(server): validated records for single text-class must include annotations

### PR DESCRIPTION
**Pull request by frascuchon**

### CODE REVIEW

File: src/rubrix/server/tasks/text_classification/api/model.py [hunk 1]
```diff
@@ -134,11 +134,27 @@ class CreationTextClassificationRecord(BaseRecord[TextClassificationAnnotation])
...
+    @classmethod
+    def _check_annotation_integrity(
+        cls,
+        annotation: TextClassificationAnnotation,
+        multi_label: bool,
+        status: TaskStatus,
+    ):
+        if status == TaskStatus.validated and not multi_label:
+            assert (
+                len(annotation.labels) > 0
```
Comment (dcfidalgo) [inline src/rubrix/server/tasks/text_classification/api/model.py]: I think this fails if `annotation is None` which could be the case I think, not sure though ...
Could you add the test case when the status is set to `validated` but `annotation is None`?

**Review by dcfidalgo — COMMENTED** [review]
:+1: Just one inline comment/doubt.

File: src/rubrix/server/tasks/text_classification/api/model.py (hunk 1 above)
Comment (frascuchon) [reply src/rubrix/server/tasks/text_classification/api/model.py]: Fixed! Thanks!

**Comment by frascuchon** [comment]
See #526

**Review by dcfidalgo — APPROVED** [review]
